_migrate-or-upgrade/rolling-upgrade/index.md

```diff
@@ -116,7 +123,15 @@ Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/migrate-or-upgrade/ro
    os-node-03  7.10.2   dimr       -
    ```
    `os-node-01` is no longer listed because the container has been stopped and deleted.
-1. Deploy a new container running the desired version of OpenSearch and mapped to the same volume as the container you deleted.
+1. Upgrade the node.
+     - If running in docker, deploy a new container running the desired version of OpenSearch, mapped to the same volume as the container you deleted.
+     - If using [Debian]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/debian/) or [RPM]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/rpm/) packages, install OpenSearch using `rpm`, `yum` or `dpkg` and start the service. No further configuration is needed as locations and files are preserved.
+     - If using [Tarball]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/tar/), the following actions are required:
+        - Copy previous `data` directory to the new `data` directory, **otherwise data will be lost**.
+        - Copy previous `opensearch.yml` file to the new `config/opensearch.yml` file.
+        - Copy previous `jvm.options` file to the new `config/jvm.options` file.
+        - Copy TLS certificates listed in `opensearch.yml` file to `./config/` directory.
```

Comment:
The current tarball instructions can be clearer about the proper sequence. The way it's written now suggests copying files after extraction, but the correct approach is to backup the files first, then extract the new tarball, then restore the backed-up files.